src/pages/Settings/SavedSearchBeta.vue

```diff
@@ -68,3 +75,28 @@ export default {
 	}
 };
 </script>
+
+<style scoped>
+main {
+	max-width: 10000px;
+	margin: 0;
+}
+
+.container {
+	padding: 2px 16px;
+}
+
+.card {
+	display: grid;
+	grid-template-columns: repeat (auto-fit, minmax(300px, 1fr));
+	grid-gap: 1rem;
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
```

Comment:
We generally don't use drop shadows in Kiva Classic. This should be a border instead. `tw-border`